src/compile/mark/path.ts

```diff
@@ -0,0 +1,42 @@
+import {X, Y, SHAPE, SIZE} from '../../channel';
+import {Config} from '../../config';
+import {ChannelDefWithLegend, FieldDef, field} from '../../fielddef';
+import {Scale} from '../../scale';
+import {VgValueRef} from '../../vega.schema';
+import {GEOJSON} from '../../type';
+
+import {applyColorAndOpacity} from '../common';
+import {UnitModel} from '../unit';
+
+
+export namespace path {
+  export function markType() {
+    return 'path';
+  }
+
+  export function properties(model: UnitModel, fixedShape?: string) {
+    // TODO Use Vega's marks properties interface
+    let p: any = {};
+    const config = model.config();
+
+    p.path = path(model.encoding().geopath);
+    p.stroke = stroke(config);
+    applyColorAndOpacity(p, model);
+    return p;
+  }
+
+  function stroke(config: Config) {
+    if (config && config.mark && config.mark.stroke) {
+      return { value: config.mark.stroke};
+    }
+    // TODO: Default stroke is white ?
```

Comment:
Default stroke should be in the config.  

Maybe add `config.mark.PathStroke`?